handlers/confirmSigHandler.go

```diff
@@ -0,0 +1,55 @@
+package handlers
+
+import (
+	"github.com/FourthState/plasma-mvp-sidechain/msgs"
+	"github.com/FourthState/plasma-mvp-sidechain/store"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// NewSpendHandler sets the inputs of a spend msg to spent and creates new
+// outputs that are added to the data store.
+func NewConfirmSigHandler(ds store.DataStore) sdk.Handler {
+	return func(ctx sdk.Context, msg sdk.Msg) sdk.Result {
+		confirmSigMsg, ok := msg.(msgs.ConfirmSigMsg)
+		if !ok {
+			panic("Msg does not implement SpendMsg")
+		}
+
+		/* Get tx, update tx data with confirm sigs, and store updated tx object */
```

Comment:
Instead of doing single comments like this. Can you break that up and comment the places where each point occurs